fdb-extensions/src/main/java/com/apple/foundationdb/async/hnsw/HNSW.java

```diff
@@ -459,8 +454,8 @@ private CompletableFuture<NodeReferenceWithDistance> greedySearchInliningLayer(@
                                     updatedNodes.put(candidateReference.getPrimaryKey(),
                                             nodeFactory.create(candidateReference.getPrimaryKey(),
                                                     baseCompactNode.getVector(), candidateNode.getNeighbors()));
-                                    return null;
-                                });
+                                })
+                                .thenApply(ignored -> null);
```

Comment:
It may be worth commenting here that this is to update the return type, as otherwise it doesn't really make sense